README.md

```diff
@@ -7,7 +7,7 @@ Topics covered:
 * Manage [App Store and Google Play](/products/appstores) content updates.
```

Comment:
I assume you don't mention Firefox for iOS because there's not enough content in that section there yet?